crates/sdk/src/env.rs

```diff
@@ -208,3 +208,13 @@ pub fn time_now() -> u64 {
 
     u64::from_le_bytes(bytes)
 }
+
+/// Call the contract's `send_proposal()` function through the bridge.
+///
+/// # Parameters
+///
+/// * `actions` - The proposal actions, sent as raw data.
+///
+pub fn send_proposal(actions: &[u8]) {
```

Comment:
this should return the `ProposalId` as well, check with Xabi what the ideal type for that is

```suggestion
pub fn send_proposal(actions: &[u8]) -> ProposalId {
```

this will be determined by the runtime (random) and mapped there

`VMLogic` should have a `Map<ProposalId, Vec<u8>>`. Which eventually, should make it to `Outcome`.

Also ideally, `ProposalId` should be constant-size.

Reply:
This is not clear to me, as I'm not sure where the proposal ID will come from. I've added the type as a `u64` for now, until the actual type is available to use, and also added a TODO... can you give me a hint as to where that ID will come from? Perhaps that will become clearer once the functionality using it is present?